util/gen_effect_id.py

```diff
@@ -51,6 +51,15 @@
     "Flourishing Fan Dance": "1820",
     "Higanbana": "1228",
     "Wildfire": "861",
+    "Chain Stratagem": "1221",
+    "Vulnerability Up": "638",
+}
+
+# These custom name of effect will not be checked.
+# Use this only when you need to handle different effects with a same name.
+# The name should be clean.
+custom_mapping = {
+    "Emboldenself": "1239",
```

Comment:
```suggestion
    "EmboldenSelf": "1239",
```

Sorry, one last comment.  Can this be `EmboldenSelf` like it was before? I think it's still two words.  <_<

Reply:
Sorry, but a clean name should only have the first character uppercase in one word. Other letter should be lower case.
If you want a finally EmboldenSelf in effect_id, you should "Embolden Self" here.

Reply:
Oh no.  I see.  Sorry for poor suggestions, then.  I think if it's a choice between `EmboldenSelf` or `Embolden (Self)` + clean or `Emboldenself` + error check, I'd prefer the first one with `EmboldenSelf` on its own.  Sorry for not remembering how the clean function works.  :C

Reply:
Well, you means directly input a clean name and not check it after?

Reply:
Yeah.  I think it's clearer to just have the name specified directly when we are making it up.

Reply:
Done.

Reply:
(I'm really sorry!! I didn't mean to go around in such a circle.  :C)